### ISSUE TITLE
Discrepancy between ability on weapon and skill learned (BLM)

### ISSUE THREAD

**Issue by FelixGrafschmidt** [TojiKitten/FFTA-randomizer#77]
I am experiencing an issue where the ability that a weapon is supposed to tach does not match the ability it actually teaches. So far I can say for certain that it effetc Nu Mou Black Mages with the standard Rod and the Firewheel Rod.
I attached my settings and will continue testing. If another mismatch comes up I will post an update.
[first.zip](https://github.com/TojiKitten/FFTA-randomizer/files/8753432/first.zip)

**Comment by FelixGrafschmidt** [comment]
It apparently also affects thieves.

**Comment by FelixGrafschmidt** [comment]
I now have a Viera Archer and a Human Archer. The Human Archer receives the abilitites it says on the weapon, the Viera Archer doesn't.
Is this intended?

**Comment by fgkeiji** [comment]
Yeah, this one is expected. You can use the "Item Log" in the randomizer itself to see the actual abilities that each item teaches.

Let's take "Short Sword" as an example. In vanilla it teaches two abilities, First Aid for Human and First Aid for Bangaa. In order to randomize abilities, the randomizer will swap the vanilla ability ID for a new ability ID. This one shows correctly for "Short Sword" because Soldier and Warrior have a different row on the item's ability menu. For classes like Black Mage, Archer, and Thief where multiple races have the same class, the game stacks those on one line and rotates the icon. This means that the in game menu won't show the actual ability learned, but it is changed in the code.

**Comment by FelixGrafschmidt** [comment]
Alright, that's basically what I expected.
Is there any lists of known quirks like this? That might spare you some issue reports.
Is it alright for me to close this issue or do you prefer to do that yourself (I know some poeple who want to keep to their specific workflow with these things, that's why I am asking)
